src/main/scala/pt/tecnico/dsi/openstack/designate/services/Recordsets.scala

```diff
@@ -0,0 +1,26 @@
+package pt.tecnico.dsi.openstack.designate.services
+
+import cats.effect.Sync
+import cats.syntax.flatMap._
+import org.http4s.client.Client
+import org.http4s.{Header, Query, Uri}
+import pt.tecnico.dsi.openstack.common.models.WithId
+import pt.tecnico.dsi.openstack.common.services.CrudService
+import pt.tecnico.dsi.openstack.designate.models.Recordset
+
+final class Recordsets[F[_]: Sync: Client](baseUri: Uri, authToken: Header)
+  extends CrudService[F, Recordset, Recordset.Create, Recordset.Update](baseUri, "recordset", authToken, wrapped = false) {
+
+  def getByName(name: String, extraHeaders: Header*): F[WithId[Recordset]] =
```

Comment:
Codacy found an issue: [Do not prefix getters with 'get'](https://app.codacy.com/gh/ist-dsi/scala-designate-client/pullRequest?prid=5821613)